packages/github/src/interfaces.ts

```diff
@@ -1,39 +1,29 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
+// TODO: https://github.com/actions/toolkit/issues/291, ESLint chokes on the | syntax
+/* eslint-disable no-undef */
+import Webhooks from '@octokit/webhooks'
+type WebhookPayload =
+  | Webhooks.WebhookPayloadPush
+  | Webhooks.WebhookPayloadPullRequest
+  | Webhooks.WebhookPayloadPullRequestReview
+  | Webhooks.WebhookPayloadPullRequestReviewComment
+  | Webhooks.WebhookPayloadStatus
+  | Webhooks.WebhookPayloadIssues
+  | Webhooks.WebhookPayloadIssueComment
+  | Webhooks.WebhookPayloadRelease
+  | Webhooks.WebhookPayloadRepositoryDispatch
+  | Webhooks.WebhookPayloadCheckRun
+  | Webhooks.WebhookPayloadDeployment
+  | Webhooks.WebhookPayloadCheckSuite
+  | Webhooks.WebhookPayloadWatch
+  | Webhooks.WebhookPayloadDeploymentStatus
+  | Webhooks.WebhookPayloadCreate
+  | Webhooks.WebhookPayloadDelete
+  | Webhooks.WebhookPayloadProjectCard
+  | Webhooks.WebhookPayloadPageBuild
+  | Webhooks.WebhookPayloadFork
+  | Webhooks.WebhookPayloadGollum
+  | Webhooks.WebhookPayloadMilestone
+  | Webhooks.WebhookPayloadProject
+  | Webhooks.WebhookPayloadLabel
 
-export interface PayloadRepository {
-  [key: string]: any
-  full_name?: string
-  name: string
-  owner: {
-    [key: string]: any
-    login: string
-    name?: string
-  }
-  html_url?: string
-}
-
-export interface WebhookPayload {
-  [key: string]: any
-  repository?: PayloadRepository
-  issue?: {
-    [key: string]: any
-    number: number
-    html_url?: string
-    body?: string
-  }
-  pull_request?: {
-    [key: string]: any
-    number: number
-    html_url?: string
-    body?: string
-  }
-  sender?: {
-    [key: string]: any
-    type: string
-  }
-  action?: string
-  installation?: {
-    id: number
-    [key: string]: any
-  }
-}
+export {Webhooks, WebhookPayload}
```

Comment:
re-export the Webhooks interfaces so consumers can reuse those via

import {Webhooks} from '@actions/github/lib/interfaces'

Reply:
do we need to re-export Webhooks? why not let users import webhooks interfaces the normal way (from `@octokit/webhooks`)